php/PhpJsonApiSamples/ClientSideEncryptionDemo1.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<!--
+To change this license header, choose License Headers in Project Properties.
+To change this template file, choose Tools | Templates
+and open the template in the editor.
+-->
+
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Client side Encryption</title>
+        
+        <!-- This page shows Demo for using PayTrace client side Encryption library - after the client side validation are met.-->
+        
+        <!-- This is the PayTrace End-to-End Encryption library: -->
+        <script language="javascript" src="https://api.paytrace.com/assets/e2ee/paytrace-e2ee.js"></script>
+        <!-- Note: PayTrace uses jquery 1.11.1 so please use the same version if possible.Using different versions of jquery can cause some issues.
+          in case of using different version of jQuery - refer this https://api.jquery.com/jquery.noconflict/-->
+        <!-- for the demo purpose, using CDN path -->
+        <script src="https://code.jquery.com/jquery-1.11.1.js"> </script>
+         
+        <script language="javascript">
+          
+                // set the key from an AJAX call  
+                $(document).ready(function(){
+                        paytrace.setKeyAjax('DemoPublic_Key.pem') ;// set the key from an AJAX call (in this case via a relative URL)
+                });
+                
+                $(document).ready( function(){
```

Comment:
For the readability, I just thought to draw developer's attention to set the key.